projects/UIP.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: project
+title: UIP
+permalink: projects/UIP
+description: A cross-platfrom desktop application for wallpapers.
```

Comment:
but i dont get what `for wallpapers` means? rather make it, `A cross-platform application for your desktop` ?